third_party/terraform/resources/resource_compute_instance.go.erb

```diff
@@ -58,6 +58,33 @@ var (
 	}
 )
 
+// network_interface.[d].network_ip can only changge when subnet/network
+// is also changing. Validate that if network_ip is changing this scenario
+// holds up to par.
+func forceNewIfNetworkIPChangedAndNotAble(ctx context.Context, d *schema.ResourceDiff, meta interface{}) error {
```

Comment:
the "and not able" here sounds like there's more words missing. Maybe something like "determineNetworkIPForceNew" would be clearer?